packages/web/src/pages/artist-dashboard-page/ArtistDashboardPage.tsx

```diff
@@ -380,9 +403,89 @@ export class ArtistDashboardPage extends Component<
     )
   }
 
+  renderUSDCSection() {
+    const { account } = this.props
+    if (!account) return null
+
+    // TODO: wire up balance https://linear.app/audius/issue/PAY-1761/wire-up-usdc-balance-in-artist-dashboard
+    const balance = 10.29
+
+    const menuItems: PopupMenuItem[] = [
+      {
+        text: messages.salesSummary,
+        // TODO: link to sales page https://linear.app/audius/issue/PAY-1763/wire-up-salespurchases-pages-on-artist-dashboard
+        onClick: () => {}
+      },
+      {
+        text: messages.withdrawHistory,
+        // TODO: link to withdraw history page https://linear.app/audius/issue/PAY-1763/wire-up-salespurchases-pages-on-artist-dashboard
+        onClick: () => {}
+      }
+    ]
+
+    return (
+      <div className={styles.usdcContainer}>
+        <div className={styles.backgroundBlueGradient}>
+          <div className={styles.usdcTitleContainer}>
+            <div className={styles.usdcTitle}>
+              {/* TODO: update icon https://linear.app/audius/issue/PAY-1764/update-icons-in-usdc-tile */}
+              <Icon icon={IconNote} size='xxxLarge' />
+              <div className={styles.usdc}>
+                <Text
+                  variant='heading'
+                  size='xxLarge'
+                  color='staticWhite'
+                  strength='strong'
+                >
+                  {messages.usdc}
+                </Text>
+              </div>
+            </div>
+            <div className={styles.usdcBalance}>
+              ${formatCurrencyBalance(balance)}
+            </div>
+          </div>
+          <div className={styles.usdcInfo}>
+            <Text color='staticWhite'>{messages.earn}</Text>
+            <HarmonyPlainButton
+              // TODO: wire up learn more link https://linear.app/audius/issue/PAY-1762/wire-up-learn-more-link
+              onClick={() => {}}
+              iconLeft={IconQuestionCircle}
+              variant={HarmonyPlainButtonType.INVERTED}
+              text={messages.learnMore}
+            />
+          </div>
+        </div>
+        <div className={styles.withdrawContainer}>
+          <HarmonyButton
+            variant={HarmonyButtonType.SECONDARY}
+            text={messages.withdraw}
+            // TODO: update leftIcon and wire up withdraw modal https://linear.app/audius/issue/PAY-1754/usdc-withdrawal-flow-ui
+            iconLeft={() => <Icon icon={IconNote} size='medium' />}
```

Comment:
Why are we not passing IconNote directly?
And if we really need the override, can we extract it to a component instead of redefining it on each render?

Reply:
i think `IconNote` might be incorrectly sized which is why i had to override. With the correct icon i'm hoping it will work without. gonna leave for now, i have a linear to update the icons once they're added to figma.